### ISSUE TITLE
[TASK] New release

### ISSUE THREAD

**Issue by fabarea** [FluidTYPO3/fluidcontent_core#228]
Hi,

Would it be possible to make a tagged / TER release of EXT:fluidcontent_core? 
Running PHP7, we have severe PHP warnings. This is however fixed on the development branch.

Thanks in advance.

**Comment by xf-** [comment]
i Try to bring it up, but after the release of fluid_styled_content this extension is more or less abandoned.

**Comment by xf-** [comment]
@NamelessCoder any comment?

**Comment by NamelessCoder** [comment]
Hi guys,

One of these days I expect `fluidcontent_core` will be abandoned and replaced with another package that integrates Flux with `fluid_styled_content`. It may mean that the controller feature gets dropped but it should not be too difficult to create a custom controller for a content element when needed. Other than that, we can continue to support putting Flux fields in template files exactly like `fluidcontent_core` does now, except in `fluid_styled_content`.

So I am not sure if it is worth while to make a new release of `fluidcontent_core`... if someone wants to prepare it I will happily do the releasing, but updating it to the most recent TYPO3 etc. probably isn't necessary if it will be abandoned soon anyway.

**Comment by fabarea** [comment]
Ok, thanks for giving a direction on this. If, on the one hand, it requires migration for us, it also means we are closer to the Core afterwards, so ok. For the time being, I guess we will fork the repo and made the adjustment there for the PHP7 path.

When abandoning this extension "officially", it would be really useful to give some hints about how to migrate to EXT:fluid_styled_content (good practices?). I guess it is not too hard but I wouldn't have a clue currently ATM. I am also wondering if it has implication about EXT:fluidpages and EXT:fluidcontent.

**Comment by kristiangoos** [comment]
Yes,  we need some directions/advices  how/if can be done some migration from fluidcontent_core to fluid_styled_content. @NamelessCoder is there any reliable and easy way doing this ? 
Currently  alot of people have sites based on fluidcontent_core  some very big with alot of content which will be deadly if everything have to be re-created ...

**Comment by NamelessCoder** [comment]
Status so far is:

* @jmverges has a half working proof of concept underway to register Providers for the standard `fluid_styled_content` elements.
* When that's completed (he needs some help from me so it is me holding it up) it should theoretically be possible to 1) override FSC TS to use different template files, i.e. the ones you made for `fluidcontent_core`, and 2) have the Flux form therein rendered and the values made available.
* There is bound to be some slight differences but we aim to make it as transparent as possible, but I'm not sure there is any good way to automate such migration (we wouldn't really know where to adjust TS without some complex migration, so a human is needed for that part - shouldn't be more than a hundreds TS lines tops for an ext that had overrides of every last core content type template).
* Beyond that, templates that were not overridden in your ext will need to be restored from FSC itself and added as overrides - because there are definitely differences in the rendering FSC does (and there are differences introduced in 8.7 as well compared to 7.6).

None of the above technically needs to have FSC installed while performing the migration but it might be nice if we create one last release compatible with 7.6 and 8.7 and mark that obsolete - just so you can install the ext and have the template files available since you most likely will need to reference those or simply use the root paths from `fluidcontent_core` as the base for your own FSC template paths.

Hopefully that explains what to expect. Sorry I can't prepare you better for this, but we don't yet know what said Provider for core's content types is going to look like (it is slightly complicated due to FSC insisting on pure TS definitions with cObjects defining the template file being used).

**Comment by NamelessCoder** [comment]
Oh and it goes without saying: if you can migrate sensibly by changing CSS/JS to adapt to FSC's output then that would be my recommendation. It would leave you in a better spot by not having to carry around legacy templates. So it is worth while to consider that as an alternative to replacing templates.

**Comment by jmverges** [comment]
Hi guys,

sorry for the late response. Yes, I started a proof of concept for integration flux in fluid_styled_content. At the moment is able to render the Flux form and save it but I need to spend some time in the controller, I will keep you informed.

**Comment by samowitsch** [comment]
Any progress here?

**Comment by jmverges** [comment]
Hi guys,

after working in 2 projects with fluid_styled_content, I have to say we shouldn't abandon this extension. I'm writing here to see if I'm the only one having the feeling fsc is far away to provide the confort fcc is providing. Any comment?

Cheers.

**Comment by samowitsch** [comment]
It seems i am the only one here who is interested in ext:fluidcontent_core 😆 @jmverges what is the state with your half running version that @NamelessCoder mentioned above? 
Or is it better to switch to FSC because fluidcontent_core now is obsolete/abandoned?
